packages/web/components/Dashboard/Calendar.tsx

```diff
@@ -247,100 +240,103 @@ export const Calendar: React.FC = () => {
-                                  location={event.location}
-                                  addToCalUrl={buildAddToCalendarLink(event)}
-                                />
-                              </Popover>
+                            return (
                               <Box
-                                position="absolute"
-                                inset={0}
+                                as="li"
+                                position="relative"
+                                className="calendar__day--event"
+                                width="100%"
+                                px={5}
+                                py={2}
+                                backgroundColor="blackAlpha.500"
+                                borderRadius="md"
+                                overflow="hidden"
+                                role="group"
                                 _groupHover={{
                                   backgroundColor: 'blackAlpha.600',
                                 }}
-                                transition="all 0.1s ease-in-out"
-                                zIndex={-1}
-                              />
-                            </Box>
-                          ))}
+                              >
+                                <Popover colorScheme="purple">
+                                  <Event
+                                    title={event.summary}
+                                    start={
+                                      'dateTime' in event.start
+                                        ? DateTime.fromISO(
+                                            event.start.dateTime,
+                                          ).toLocaleString(
+                                            DateTime.TIME_24_SIMPLE,
+                                          )
+                                        : DateTime.fromISO(
+                                            event.start.date,
+                                          ).toLocaleString(
+                                            DateTime.TIME_24_SIMPLE,
+                                          )
+                                    }
+                                    end={
+                                      'dateTime' in event.end
+                                        ? DateTime.fromISO(
+                                            event.end.dateTime,
+                                          ).toLocaleString(
+                                            DateTime.TIME_24_SIMPLE,
+                                          )
+                                        : DateTime.fromISO(
+                                            event.end.date,
+                                          ).toLocaleString(
+                                            DateTime.TIME_24_SIMPLE,
+                                          )
+                                    }
```

Comment:
Same question as above:

```suggestion
                                    end={
                                      DateTime.fromISO(
                                        event.end.dateTime ?? event.end.date,
                                      ).toLocaleString(
                                        DateTime.TIME_24_SIMPLE,
                                      )
                                    }
```

Reply:
replied above